data/remnant missions.txt

```diff
@@ -1,497 +1,1358 @@
+		personality staying heroic nemesis uninterested
+		ship "Archon (Cloaked)" "Lifted Lorax"
+	on stopover
+		conversation
+			`You have dodged the Archon and landed on the second of the two gas giants where the void sprites live. The hull of your tiny ship creaks and groans under the high atmospheric pressure, but seems to be holding together. Its sensors pick up flocks of void sprites floating between the planet's cloud layers, presumably feeding on airborne creatures too small to detect from this range.`
+			`	You also observe some void sprites leaving the planet, soaring upward on thermal updrafts. As they rise, their bodies swell to several times their original size, like an aerostat balloon gaining altitude. But clearly they have another form of propulsion as well, something akin to the antigravity repulsors that human ships use to travel from a planet's surface into orbit.`
+			`	After collecting some atmospheric samples for good measure, you prepare to return to <planet>. Hopefully the Archon will calm down once it sees that you have not attacked or disturbed the Sprites on either of these planets.`
+	on complete
+		event "remnant: void sprite research" 20
+		log "Factions" "Void Sprites" `The "void sprites" are space-dwelling lifeforms. They are apparently not sentient. They evolved in the atmosphere of a gas giant, but have some biological mechanism for altering or reflecting gravity that allows them to fly into outer space, in the same way that human ships use antigravity to escape a planet's gravity well. The void sprites are not particularly powerful or dangerous creatures, but they are guarded by a Drak Archon.`
+		log "Factions" "Archons" `Archons are partly biological, but do not seem to be naturally evolved creatures. Some of their biological characteristics may have been based on the "void sprites" that inhabit the Ember Waste.`
+		conversation
+			`The Remnant researcher is ecstatic when he sees all the data you collected. "I will begin assembling this into a research paper immediately!" he sings. "It may be months before it is published, but when it is you will be named as a co-author, and I will share the payment with you."`
+			choice
+				`	"What do you know about the Archon that attacked me?"`
+				`	"I'm glad I could help you out. I'll look forward to hearing from you."`
+					decline
+
+			`	He begins singing you a song about the Archons. The song says that the Archons are servants of the mysterious Drak, whom no one has seen but whom the Quarg claim to speak to and take orders from. "That's all our histories say," he sings, "but what interests me is the biological similarities between the Archons and the void sprites. We do not think the Archons are naturally evolved creatures. Perhaps their creators incorporated elements of void sprite biology into them when they were made. Or perhaps the Archons just feel an affinity with them because they are both space-faring organisms."`
+			choice
+				`	"Well, I'll look forward
```

Comment:
If I'm not wrong, the Quarg are unwilling to trade technology with humanity, especially the Jump Drives. Or maybe they only give away non-military technology (except the Jump Drive).

Reply:
I'm not sure if it is explicitly stated anywhere in terms of technology, but Tarazed shipyards provides some indirect evidence: They're headquartered right next door to a Quarg world and several Quarg systems, with Quarg ships and the Quarg themselves being a fairly common sight. The shipyard descriptions even mention that some of their designs incorporate elements inspired by alien technology. And despite all that, there's nothing really revolutionary about Tarazed's designs. They aren't massively superior in the way that one would expect if they actually had access to Quarg tech. 

This is where I get my assumption that the Quarg are fine with people watching them, looking at them, probably even scanning and trying to imitate them. If they weren't, they probably wouldn't be very permissive of having a major shipyard and R&D center located right next door to them. A shipyard R&D center is virtually guaranteed to have tons of scanners everywhere. If nothing else, they would be incredibly useful for testing new prototypes. And developers being who they are, it is virtually guaranteed that most of them would probably turn those scanners on visiting alien ships at every opportunity. (Maybe Quarg ships need some built in interference plating?)

Actually, it is stated in the Rim Archeology missions that Quarg are relatively free with information, so it may be a case of them being generous in sharing what one might term "pure knowledge" but aren't giving away anything in terms of actual tech. Sort of a "We'll teach you the laws of the universe, but it is up to you to figure out how to use them" sort of thing.